pkg/controllers/repositoryserver/utils.go

```diff
@@ -248,3 +249,12 @@ func getVolumes(ctx context.Context, cli kubernetes.Interface, secret *corev1.Se
 	}
 	return vols, nil
 }
+
+func GetIntOrDefault(value string, defaultValue int) (int, error) {
```

Comment:
There are similar utilities in pkg/utils/utils.go, should we move this utility as well there?